wordpress-mobile/WordPress-iOS
# Update gutenberg bridge to support MediaInfo

Updates Gutenberg project and updates code to support MediaInfo struct to pass around media picker data.

To test:
 - Start the app
 - Create a new post using the block editor
 - Add image and video blocks check that the different sources work correctly
 - Try to copy one image from the photos app and paste inside a text block, check that an image block is added and you see an image upload simulation happening.

Update release notes:
